diff-containers/changelog.d/scriv.ini

```diff
@@ -0,0 +1,13 @@
+[scriv]
+format = md
+insert_marker = Changelog entries
```

Comment:
This was used to ignore the header on every changelog and only start when this string is found. I think you should remove it in this case

Reply:
I've commented them out for now